### DISCUSSION TITLE
The copilot cannot be used after purchase

### DISCUSSION THREAD

**Discussion by DragonUncle** [community/community#45746] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

The copilot cannot be used after purchase

[github-DragonUncle-receipt-2023-01-05.pdf](https://github.com/community/community/files/10531380/github-DragonUncle-receipt-2023-01-05.pdf)
![1675041299309](https://user-images.githubusercontent.com/55368916/215368247-b8d8033d-dccc-4640-a874-e1c858ce577e.jpg)

**Comment by davecheney** [comment, marked as answer]
@DragonUncle please contact support, https://support.github.com/contact
